tools/ansible-pr/run.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+set -x
+
+declare -a ignored_modules=(
+  gcp_backend_service
+  gcp_forwarding_rule
+  gcp_healthcheck
+  gcp_target_proxy
+  gcp_url_map
+)
+
+new_branch() {
+  branch_name=$1
+  git  checkout upstream/devel
+  git branch -D $branch_name
+  git chkecout -b $branch_name
```

Comment:
`git chkecout` are you sure this is getting called (and `set -e` would catch this :-)